src/CSET/operators/plot.py

```diff
@@ -948,20 +948,21 @@ def spatial_pcolormesh_plot(
 #     Coordinate about which to plot multiple lines. Defaults to
 #     ``"realization"``.
 def plot_line_series(
-    cube: iris.cube.Cube,
+    toplot: iris.cube.Cube | iris.cube.CubeList,
```

Comment:
I don't like these runtogetherwords, as I read that as "top lot". Python code tends to use snake_case, where the words are separated with underscores.

The operators we already have that take either a Cube or a CubeList call this first arguments `cubes`, so perhaps we should continue that.